force-app/main/default/lwc/givingSummary/givingSummary.html

```diff
@@ -0,0 +1,77 @@
+<template>
+  <lightning-card>
+    <lightning-layout multiple-rows="true" horizontal-align="center">
+      <lightning-layout-item size="12">
+        <h2 class="slds-text-heading_small slds-var-p-left_large">
+          {labels.donorsGivingSummary}
+        </h2>
+      </lightning-layout-item>
+      <template if:true={errorMessage}>
+        <lightning-layout-item
+        size="12"
+        small-device-size="12"
+        medium-device-size="12"
+        large-device-size="12"
+        class={rowClasses}
+      >
+        <span class="slds-var-m-top_medium bolder">{labels.commonUnknownError}</span>
+      </lightning-layout-item>
+      </template>
+      <template if:false={errorMessage}>
+        <lightning-layout-item
+          size="12"
+          small-device-size="12"
+          medium-device-size="4"
+          large-device-size="4"
+          class={rowClasses}
+        >
+          <span class="bolder">
+            <lightning-formatted-number
+              class="lifetime"
+              value={lifetimeSummary}
+              format-style="currency"
+              currency-code="USD"
+            >
+            </lightning-formatted-number> </span
+          ><br />
+          <span class="slds-var-m-top_medium">{labels.donorsLifetime}</span>
```

Comment:
The label api names in this file need to be updated to match the new label api names

Reply:
done